### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_AzureNotebooksWelcome_Inspect: Alt text missing for "Azure Notebook" image present at header pane.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3937]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106 
Operating System:Windows10+

**Repro Steps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to run button & hit enter --> Navigate to Azure Notebooks welcome & hit enter.
3. Verify alt text for images or logos present on page.

**Actual:**
Alternative text is missing for "Azure Notebook" image present at header pane.
Same issue observed on all pages where "Azure Notebook" image available in header pane.

**Expected:**
Images or logos should have alternative text.

**User Impact:**
Screen reader user would not get information about images present on page.

Attachment:
![a11y_azurenotebooks_jupyter_azurenotebookswelcome_inspect](https://user-images.githubusercontent.com/39204909/45751296-955a2d80-bc2f-11e8-861f-203d6fe7bb31.jpg)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
